Sources/audio-devices/AudioDevices.swift

```diff
@@ -37,6 +38,50 @@ struct AudioDevice: Hashable, Codable, Identifiable {
 
     uid = deviceUID as String
 
+    var deviceTransportType: UInt32 = 0
+    do {
+      try CoreAudioData.get(
+        id: deviceId,
+        selector: kAudioDevicePropertyTransportType,
+        value: &deviceTransportType
+      )
+    } catch {
+      deviceTransportType = 0
+    }
+
+    switch deviceTransportType {
+      case kAudioDeviceTransportTypeBluetooth,
+           kAudioDeviceTransportTypeBluetoothLE:
```

Comment:
These should be individual types. It should be up to the consumer to decide what to do about them.